source/reference/command/flushRouterConfig.txt

```diff
@@ -12,13 +12,13 @@ flushRouterConfig
 
 .. dbcommand:: flushRouterConfig
 
-   :dbcommand:`flushRouterConfig` clears the current cluster
-   information cached by a :binary:`~bin.mongos` instance and reloads all
-   :term:`sharded cluster` metadata from the :term:`config database`.
+   :dbcommand:`flushRouterConfig` clears the cached routing table in a
+   :binary:`~bin.mongos` instance. Subsequent :binary:`~bin.mongos`
+   commands update the routing table as routes are needed.
 
-   This forces an update when the configuration database holds data
-   that is newer than the data cached in the :binary:`~bin.mongos`
-   process.
+   When this command is called, it forces an update when the
+   configuration database holds data that is more recent than the data
+   cached in the :binary:`~bin.mongos` process.
```

Comment:
This sounds a little awkward. How about:

Use this command to force the routing table cache to be refreshed from the config server when there is possibility that it contains a more recent data. In almost all cases, this happens automatically, so it typically only needs to be done after a movePrimary as per instructions.